xAPI.md

```diff
@@ -438,6 +439,16 @@ __Registration__: An instance of a learner experiencing a particular Activity.
 __Representational State Transfer (REST)__: An architecture for designing networked web Services.
 It relies on HTTP methods and uses current web best practices.
 
+<a name="def-reporting-system" />
+
+__Reporting System__: A system or tool used to report on and analyse xAPI data and display this to the user. 
```

Comment:
More touching on that many reporting systems won't display things to the user themselves, they'll forward the data elsewhere. And if the requirements are for displaying things to someone, that should probably be explicit in the requirements, instead of implicit in an aspect of the Reporting System definition.